packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxRipple from "./OnyxRipple.vue";
+
+/**
+ * OnyxRipple provides visual feedback when a user interacts with the component. Supporting both touch and mouse events.
+ */
+const meta: Meta<typeof OnyxRipple> = {
+  title: "Support/OnyxRipple",
```

Comment:
```suggestion
  title: "Support/Ripple",
```

Also the example preview looks quiet empty, maybe we can add a button/text or similar to showcase it?

Reply:
added an generic example via template